Chatto/sources/ChatController/BaseChatViewController.swift

```diff
@@ -107,6 +108,16 @@ open class BaseChatViewController: UIViewController,
         didSet { self.updateInputContainerBottomConstraint() }
     }
 
+    private func makeKeyboardUpdatesHandler() -> KeyboardUpdatesHandler {
```

Comment:
What do you think about passing `KeyboardUpdatesHandler` as a dependency?

It seems like `BaseChatViewController` has property `keyboardEventsHandlers` only to notify them whenever `KeyboardUpdatesHandler` triggers an even.

If `KeyboardUpdatesHandler` called those `keyboardEventsHandlers` directly, we could remove both `keyboardTracker` and `keyboardEventsHandlers` from the `BaseChatViewController`

Reply:
That's a good point! Even if `BaseChatViewController` still needs to add itself as the `keyboardInputAdjustableViewController` and `delegate` we can definitely do that!